ci/docker/os-image-stemcell-builder-bionic/Dockerfile

```diff
@@ -24,25 +24,47 @@ RUN apt-get update \
     lsof \
   && apt-get clean
 
+# Nokogiri dependencies
+RUN apt-get install -y libxslt-dev libxml2-dev && apt-get clean
+
 RUN groupadd -o -g ${GROUP_ID} ubuntu                  \
   && useradd -u ${USER_ID} -g ${GROUP_ID} -m ubuntu \
   && echo 'ubuntu ALL=NOPASSWD:ALL' >> /etc/sudoers
 
-# Nokogiri dependencies
-RUN apt-get install -y libxslt-dev libxml2-dev && apt-get clean
+# install ruby
+RUN \
+  wget -O ruby-install-0.8.3.tar.gz https://github.com/postmodern/ruby-install/archive/v0.8.3.tar.gz \
+  && tar -xzvf ruby-install-0.8.3.tar.gz \
+  && cd ruby-install-0.8.3/ \
+  && make install \
+  && cd .. \
+  && rm -r ruby-install* \
+  && ruby-install 3.0.2
+
+# we install chruby to allow us to run `bundle` as a non-root user
+RUN \
+  wget -O chruby-0.3.9.tar.gz https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz \
+  && tar -xzvf chruby-0.3.9.tar.gz \
+  && cd chruby-0.3.9/ \
+  && make install \
+  && cd .. \
+  && rm -r chruby-0.3.9* \
+  && echo 'source /usr/local/share/chruby/chruby.sh; chruby 3.0.2' > ~ubuntu/.bashrc
 
-ADD install-ruby.sh /tmp/install-ruby.sh
-RUN chmod a+x /tmp/install-ruby.sh
-RUN cd /tmp && ./install-ruby.sh && rm install-ruby.sh
+# ADD install-ruby.sh /tmp/install-ruby.sh
+# RUN chmod a+x /tmp/install-ruby.sh
+# RUN cd /tmp && ./install-ruby.sh && rm install-ruby.sh
 
```

Comment:
Should we remove the [install-ruby.sh](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-jammy/master/ci/docker/os-image-stemcell-builder/install-ruby.sh)?

Reply:
yes we should remove this

Reply:
Ok, let's remove it with this pr?